package.json

```diff
@@ -12,8 +12,8 @@
     "test": "npm run lint && npm run test:unit",
     "prettier": "prettier --config prettier.config.js --write 'src/**/*.{ts,tsx}'",
     "build:bundle": "cross-env NODE_ENV=production webpack -p",
-    "build:es5": "tsc -t es5 --outDir dist/es5 --declaration --inlineSourceMap --inlineSources",
-    "build:es6": "tsc -t es6 -m es6 --outDir dist/es6 --declaration --inlineSourceMap --inlineSources",
+    "build:es5": "node_modules/typescript-plugin-inner-jsx/bin/build -t es5 --outDir dist/es5 --declaration --inlineSourceMap --inlineSources --config ./tsconfig.json",
```

Comment:
Hm if you properly register it you can just call something like `tsc-transform-jsx`  (the name is up to you)

Reply:
Please see, e.g., https://medium.com/netscape/a-guide-to-create-a-nodejs-command-line-package-c2166ad0452e (section 3) for how to properly register a script in package.json (this is missing in typescript-plugin-inner-jsx).